### PR TITLE
Update docs for deleting account

### PR DESCRIPTION
**Pull request by glokta1**

### CODE REVIEW

File: docs/account/delete-account.mdx [hunk 1]
```diff
@@ -3,17 +3,19 @@ title: "How do I delete my account?"
...
-  **To**: hello@codecrafters.io
-
-**Subject**: Account deletion request
-
-**Body**:
-
-Please delete my CodeCrafters account. My GitHub username is `your_github_username`.
-
-</Card>
+1. Click on your username on the top right corner of the page.
```
Comment (rohitpaulk) [inline docs/account/delete-account.mdx:8]: Maybe we can make this simpler: 

> Visit your [account settings page](https://app.codecrafters.io/settings/account)

Comment (rohitpaulk) [reply docs/account/delete-account.mdx:8]: And the other things we can cover can be like FAQ - what happens when you delete your account, can you signup again etc?

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Notes added

**Comment by glokta1** [comment]
Small recommendation: Add a grace period during which a user can recover their account

Came to my mind while writing the FAQs

File: docs/img/delete-account/account-tab.png
Comment (rohitpaulk) [inline docs/img/delete-account/account-tab.png:1]: Don't think we're using this btw, can be deleted?

**Review by rohitpaulk — APPROVED** [review]

**Pull request merged**